lib/twiglet/validation_schema.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "object",
+  "required": ["message"],
+  "properties": {
+    "message": {
```

Comment:
Were we thinking of adding more stuff here?

Reply:
I'm open to suggestion on what else to include! This covers all the validation we do currently, though. This can/will also be overriden by a custom schema, and if it is permissive-by-design I'm not sure what else we would want/need to include.